reana_cluster/backends/kubernetes/k8s.py

```diff
@@ -606,6 +607,7 @@ def verify_components(self):
                                          deployed_img))
 
                     if not spec_img == deployed_img:
+                        all_components_match = False
```

Comment:
I will change this to append all component names that are ok and print it out as:
```
reana-server ... OK
reana-workflow-monitor ... ERROR
```